src/components/TractionMessage.vue

```diff
@@ -22,23 +49,47 @@ export default {
         return 'error-message'
       },
     },
+    /**Property to specify alert type - 'success', 'error' and 'danger' are the variants supported which are bootstrap compatible*/
     type: {
       type: String,
       default: 'primary',
     },
+    /**Message to display */
     message: {
       type: String,
       required: true,
     },
   },
   computed: {
+    /**
+     * This can be removed once enable_custom_alert feature flag is removed
+     */
     variant() {
       return (
         {
           error: 'danger',
         }[this.type] || this.type
       )
     },
+    /**
+     * Returns tailwind colors based on 'type' property with default being grey.
+     *  The supported values for 'type' property are as follows
+     * 'success' -  will be displayed in green color
+     * 'error'  -  will be displayed in red color
+     */
+    color() {
+      let value = { message: 'bg-gray-300 text-gray-400', icon: '' }
+      if (this.type === 'success') {
```

Comment:
Might be nice to have all the types as an object or something where the keys are the type. Then the computed can just search the object and return the value or fallback onto the default.

Reply:
Sure, this seems like it will make it more confined. will try 👍